tfx_addons/firebase_publisher/README.md

```diff
@@ -7,10 +7,10 @@
 
 **Your company/organization:** Individual(ML GDE)
 
-**Project name:** [Firebase ML Publisher](https://github.com/tensorflow/tfx-addons/issues/59)
+**Project name:** [FirebasePublisher](https://github.com/tensorflow/tfx-addons/issues/59)
 
 ## Project Description
-This project defines a custom TFX component to publish/update ML models from TFX Pusher to [Firebase ML](https://firebase.google.com/products/ml). The input model from TFX Pusher is assumed to be a TFLite format.
+This project defines a custom TFX component to publish/update ML models to [Firebase ML](https://firebase.google.com/products/ml). 
```

Comment:
Maybe add a line to denote why pushing models to Firebase ML is useful?

Reply:
That is noted in the `Project Use-Case(s)` section :)